backend/LexBoxApi/Jobs/DeleteTempDirectoryJob.cs

```diff
@@ -0,0 +1,30 @@
+﻿using Quartz;
+
+namespace LexBoxApi.Jobs;
+
+public class DeleteTempDirectoryJob() : DelayedLexJob
+{
+    public static async Task Queue(ISchedulerFactory schedulerFactory,
+        string path,
+        TimeSpan delay,
+        CancellationToken cancellationToken = default)
+    {
+        await QueueJob(schedulerFactory,
+            Key,
+            new JobDataMap { { nameof(Path), path } },
+            delay,
+            cancellationToken);
+    }
+
+    public static JobKey Key { get; } = new(nameof(DeleteTempDirectoryJob), "CleanupJobs");
+    public string? Path { get; set; }
+
+    protected override Task ExecuteJob(IJobExecutionContext context)
+    {
+        ArgumentException.ThrowIfNullOrEmpty(Path);
+        return Task.Run(() =>
```

Comment:
this doesn't need to be wrapped in a task since it's not running as part of a web request

Reply:
Done in commit 1e5e8169.